examples/burial-form/ReviewPage.jsx

```diff
@@ -0,0 +1,345 @@
+        pageUrl: '/military-history/previous-names',
+        fields: {
+          "previousNames" : {
+            label: "Did the Veteran serve under another name?", 
+            value: {
+              first: {
+                label: "First",
+                value: state.values?.previousNames[0]?.first
+              },
+              middle: {
+                label: "Middle",
+                value: state.values?.previousNames[0]?.middle
+              },
+              last: {
+                label: "Last",
+                value: state.values?.previousNames[0]?.last
+              }
+            }
+          }
+        }
+      },
+      {
+        title: 'Benefits Selection',
+        id: 'benefits-selection',
+        pageUrl: '/benefits/selection',
+        fields: {
+          "burialAllowance": {
+            label: "Burial Allowance",
+            value: state?.values?.benefitsSelection?.burialAllowance
+          },
+          "plotAllowance": {
+            label: "Plot or interment allowance",
+            value: state?.values?.benefitsSelection?.plotAllowance
+          },
+          "transportation": {
+            label: "Transportation expenses",
+            value: state?.values?.benefitsSelection?.transportation
+          },
+          "amountIncurred": {
+            label: "Transportation amount incurred",
+            value: state?.values?.amountIncurred
+          }
+        },
+      },
+      {
+        title: 'Benefits Selection: Type of Burial Allowance',
+        id: 'benefits-burial-allowance',
+        pageUrl: '/benefits/burial-allowance',
+        fields: {
+          "burialAllowanceRequested": {
+            label: "Type of Burial allowance requested",
+            value: state?.values?.burialAllowanceRequested
+          }
+        },
+      },
+      {
+        title: 'Benefits Selection: Plot or interment allowance',
+        id: 'benefits-plot-allowance',
+        pageUrl: '/benefits/plot-allowance',
+        fields: {
+          "placeOfRemains": {
+            label: "Place of burial or deceased Veteran’s remains",
+            value: state?.values?.placeOfRemains
+          },
+          "federalCemetery": {
+            label: "Was the Veteran buried in a national cemetary, or one owned by the federal government?",
+            value: state?.values?.federalCemetery
+          },
+          "stateCemetery":{
+            label: "Was the Veteran buried in a state veteran’s cemetary?",
+            value: state?.values?.stateCemetery
+          },
+          "govtContributions": {
+            label: "Did a federal/state government or the Veteran’s employer contribute to the burial?  (Not including employer life insurance)",
+            value: state?.values?.govtContributions
+          },
+          "amountGovtContribution": {
+            label: "Amount of government or employer contribution",
+            value: state?.values?.amountGovtContribution
+          }
+        },
+      },
+      // {
```

Comment:
Does this need to be commented out because the page isn't there to navigate back to?

Reply:
I can go ahead and uncomment this, so when the page is available, the data shows up in the review page.